subprojects/gradle/impact-shared/src/main/java/com/avito/impact/configuration/BaseConfiguration.kt

```diff
@@ -18,9 +18,9 @@ import java.io.File
 /**
  * Wrapper above [org.gradle.api.artifacts.Configuration] to reduce an amount of configurations
  */
-abstract class SimpleConfiguration(
+abstract class BaseConfiguration(
     val module: InternalModule,
-    val type: ConfigurationType
+    val types: Set<Class<out ConfigurationType>>
 ) : Equality {
 
     abstract val fullBytecodeSets: Set<File>
```

Comment:
We can remove `fullBytecodeSets`

https://avito-tech.github.io/avito-android/blog/2021/01/ui-impact-analysis-end